images_config/ks-robot-master.cfg

```diff
@@ -259,9 +262,13 @@ chmod +x /usr/sbin/cob-command
 sh -c 'echo "%users ALL=NOPASSWD:/usr/sbin/cob-command"' | sed -i -e "\|%users ALL=NOPASSWD:/usr/sbin/cob-command|h; \${x;s|%users ALL=NOPASSWD:/usr/sbin/cob-command||;{g;t};a\\" -e "%users ALL=NOPASSWD:/usr/sbin/cob-command" -e "}" /etc/sudoers 
 %end
 
-################ INSTALL APT-CACHER AND PURGE MODEMMANAGER ################ 
+################ INSTALL APT-CACHER ################ 
 %post --interpreter /bin/bash
 apt-get install apt-cacher-ng -y
-apt-get purge modemmanager -y
+
+echo 'Acquire::http { Proxy "http://server_ip:3142"; };' >> /etc/apt/apt.conf.d/01proxy
+HOSTNAME=$(cat /etc/hostname)
+SERVERNAME=$(echo ${HOSTNAME%-*}-b1)
+sed -i "s/server_ip/${SERVERNAME}/g" /etc/apt/apt.conf.d/01proxy
```

Comment:
are you sure you want to call these lines here and not with the post-install script?? if the network is not setup or base pc not connected you will not be able to call `apt-get` again

Reply:
I think that's fine. it's the very last step of the install script. after that we should assemble a set of pcs including a base pc and a router before running the postinstall script. so no need to install further apt-packages manually